### PR TITLE
allow certs to be provided as strings or file path

### PR DESCRIPTION
**Pull request by dvdliao**
to allow for more use cases, such as if cert was stored in vault

### CODE REVIEW

**Comment by dvdliao** [comment]
hey @Mongey any idea when you can get around to this? 🙂

File: kafka/provider.go [hunk 1]
```diff
@@ -17,19 +17,19 @@ func Provider() terraform.ResourceProvider {
 				Required:    true,
 				Description: "A list of kafka brokers",
 			},
-			"ca_cert_file": &schema.Schema{
+			"ca_cert": &schema.Schema{
 				Type:        schema.TypeString,
 				Optional:    true,
 				DefaultFunc: schema.EnvDefaultFunc("KAFKA_CA_CERT", ""),
 				Description: "Path to a CA certificate file to validate the server's certificate.",
```
Comment (Mongey) [inline kafka/provider.go]: Can you update these descriptions to remove reference to path / file. 🙏

**Comment by dvdliao** [comment]
👍 thanks, done. Would be good to highlight in the release for the parameter renaming.

**Comment by Mongey** [comment]
It looks like Terraform has some built in ways to help with the deprecation of attributes. https://www.terraform.io/docs/extend/best-practices/deprecations.html#provider-attribute-removal

**Comment by dvdliao** [comment]
nice, implemented an easy patch

**Pull request merged**